extensions/BeatBlox/index.js

```diff
@@ -534,15 +539,21 @@
                             appliedEffects = [];
                         }, { args: [], timeout: I32_MAX });
                     }),
+                    new Extension.Block('makeTempo','command','music','set tempo %n bpm', [120], function(tempo){
+                        audioAPI.updateTempo(4,tempo,4,4);
+                    }),
                     new Extension.Block('appliedEffects', 'reporter', 'music', 'applied effects', [], function () {
-                        if(appliedEffects.length === 0) return 'No effects applied';
+                        if(appliedEffects.length === 0) {
+                            var twoD = []        
+                            twoD = new List(twoD.map(a => new List(a)));
+                            return twoD;}
                         
                         const trackName = this.id;     
                         return getEffectValues(trackName,appliedEffects);
                     }).for(SpriteMorph,StageMorph),
                     new Extension.Block('tempo', 'reporter', 'music', 'tempo', [], function () {
                         var tempoObject = audioAPI.getTempo();
-                        return tempoObject.beatsPerMinute + ' BPM';
+                        return tempoObject.beatsPerMinute + ' BPM ';
```

Comment:
Not sure why the space was added at the end, but separate issue: this should probably just return a number, as otherwise it can't be used as the input to the new set tempo block (or any math blocks)